### ISSUE TITLE
Implicit Wav parameters

### ISSUE THREAD

**Issue by ThomasHezard** [Mercandj/audio-samples#2]
https://github.com/Mercandj/audio-samples/blob/866deecc36a769ca898b407dbfee5e88221bbae4/create_audio_samples.sh#L26

Similar issue as https://github.com/Mercandj/audio-samples/issues/1

ffmpeg's wav encoder uses 16-bits encoding by default.

And, by the way, FLAC, like all compressing formats, doesn't have the notion of bit depth from PCM files, so the 24-bits in the filename must come from the original record, but doesn't mean anything for the FLAC file.

**Comment by ThomasHezard** [comment]
several solutions to control the bit depth in a wav file :
- using `-sample_fmt` (see `ffmpeg -sample_fmts`)
- or using `-acodec pcm_XXX` (see `ffmpeg -codecs | grep ' pcm_'`)

**Comment by Mercandj** [comment]
You're right, https://superuser.com/questions/675342/convert-mp3-to-wav-using-ffmpeg-for-vbr/675647
